examples/nginx-grafana-metabase/README.md

```diff
@@ -0,0 +1,79 @@
+# Visualizing Nginx Access Logs using Grafana and Metabase
+In my last [blog post](https://www.timeplus.com/post/analyzing-nginx-access-logs), I showed how Timeplus Proton can be used to analyze web traffic in production using just the log files generated by an Nginx web server. The article explored real-time analysis and historical analysis but the analysis wasn't visualized. 
+
+This article will remedy the lack of rich visualizations by showing how to do so using open source tools like [Grafana](https://grafana.com/) and [Metabase](https://www.metabase.com/).
+
+
+## Prerequisities
+In addition to Timeplus Proton, the previous article used the following technologies: 
+* [Amazon Virtual Private Cloud](https://aws.amazon.com/vpc/) (VPC);
+* [Amazon Elastic Compute Cloud](https://aws.amazon.com/ec2/) (EC2);
+* [NFS](https://en.wikipedia.org/wiki/Network_File_System);
+* [IPinfo](https://ipinfo.io/);
+* [Ghost](https://ghost.org/);
+* [Nginx](https://nginx.org/en/).
+  
+The first two require access to an AWS account. The last two products need to have been running in production, in order to have meaningful access log data to experiment with. 
+
+This article does away with most of those prerequisities making it easier to get started. It builds on the same analysis of the previous article but you only need: 
+* [Docker](https://docs.docker.com/engine/install/) installed;
+* an [IPinfo](https://ipinfo.io/) account. 
+
+Timeplus Proton has a handy feature [`RANDOM STREAM`](https://docs.timeplus.com/proton-create-stream#create-random-stream) which we will use to generate all the access log data that we will use for experimentation in this article.
+
+Let's dive in!
+
+## Docker Environment Overview
+The [`docker-compose.yaml`](docker-compose.yaml) file defines 3 containers:
+* a Timeplus Proton container;
+* a Grafana container;
+* a third container that seeds the Timeplus Proton container with access log data.
+
+The 3rd container uses the `proton client` to execute an SQL file [`01_nginx_access_log.sql`](01_nginx_access_log.sql) used to seed Timeplus Proton with access log. That SQL file uses the [`CREATE RANDOM STREAM`](https://docs.timeplus.com/proton-create-stream#create-random-stream) DDL to create a stream named `nginx_access_log`. 
```

Comment:
I would suggest showing the main DDL of the random stream, don't have to show all settings, but give the reader an idea of how the random stream is defined without looking into the sql file (also when we publish the blog on our website, the sql file will remain in github proton repo. I guess many ppl will be too lazy to check the link. For important information, please put in the blog content directly)

Reply:
I decided to include a screenshot of the DDL so I could do a line-by-line explanation of each database function used to generate random data **and** also link out to the relevant docs.